README.md

```diff
@@ -130,20 +132,20 @@ promises. The static methods below are static methods of this constructor.
 | `p.[[ApplyFunctionSend]](args)`    | `applyFunction(p, args)` |
 | `p.[[ApplyMethodSend]](prop, args)`| `applyMethod(p, prop, args)` |
 
-The static methods first do the equivalent of `HandledPromise.resolve` on their first
+The static methods first do a `Promise.resolve` on their first
 argument, to coerce it to a promise with these internal methods.  Thus, for
 example,
 
 ```js
-HandledPromise.get(p, prop)
+Promise.get(p, prop)
 ```
 actual does the equivalent of
 ```
-HandledPromise.resolve(p).[[GetSend]](prop)
+Promise.resolve(p).[[GetSend]](prop)
```

Comment:
is this also true on a subclass of Promise? (that it uses the internal slot, i mean)

Reply:
@erights will have to answer.

Reply:
(First, I can't stop myself from saying it: I hate promise subclassing. It was never a sensible manner to extend the behavior of promises. But it's too late, so we need comprehensive answers. The following answers are the first time I've given subclassing much thought, so feedback especially appreciated.)

Given

```js
class FooPromise extends Promise {
  // overrides nothing. Inherits everything.
}
```
I would expect
```
FooPromise.get(p, prop)
```
to be equivalent to
```js
FooPromise.resolve(p).[[GetSend]](prop)
```
Since the base class is still `Promise`, an instance of `FooPromise` should still have all the exotic internal methods of a promise.